guides/v2.1/architecture/archi_perspectives/components/modules/mod_depend_types.md

```diff
@@ -1,62 +1,50 @@
 ---
 group: arch-guide
-subgroup: Components
 title: Module dependency types
-menu_title: Module dependency types
-menu_order: 7
-level3_menu_node: level3child
-level3_subgroup: modules
 version: 2.1
 github_link: architecture/archi_perspectives/components/modules/mod_depend_types.md
 redirect_from: /guides/v1.0/architecture/modules/mod_depend_types.html
 ---
 
 ## Two types of dependencies {#m2devgde-moddep-declare-dep}
 
-There are two types of Magento {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} dependencies: hard and soft dependencies.
+There are two types of Magento {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} dependencies: hard and soft.
 
-## Hard dependencies
-
-Modules with a *hard dependency* on another module cannot function without the module it depends on. Specifically:
-
-* The module contains code that directly uses logic from another module  (for example, the latter module's instances, class constants, static methods, public class properties, interfaces, and traits).
-
-* The module contains strings that include class names, method names, class constants, class properties, interfaces, and traits from another module.
+Magento installs modules in the following order:
 
-* The module deserializes an object declared in another module.
+1. The module serving as a dependency for another module
+2. The module dependent on it
 
-* The module uses or modifies the database tables used by another module.
+### Hard dependencies
 
-## Soft dependencies
+Modules with a *hard dependency* on another module cannot function without the module it depends on. These modules:
 
-Modules with a *soft dependency* on another module can function properly without the other module, even if it has a dependency upon it. Specifically:
+* Contain code that directly uses logic from another module, as for the latter module's instances, class constants, static methods, public class properties, interfaces, and traits.
+* Contain strings that include class names, method names, class constants, class properties, interfaces, and traits from another module.
+* Deserializes an object declared in another module.
+* Uses or modifies the database tables used by another module.
 
-* The module directly checks another module's availability.
+### Soft dependencies
 
-* The module extends another module's configuration.
+Modules with a *soft dependency* on another module can function properly without the other module, even if they have a dependency upon it. These modules:
```

Comment:
Consider singular version "**A module** with a soft dependency...". Regardless, omit: "even if they have a dependency upon it."

Reply:
👍